x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextInferenceResultFieldMapper.java

```diff
@@ -107,15 +110,11 @@
 public class SemanticTextInferenceResultFieldMapper extends MetadataFieldMapper {
     public static final String CONTENT_TYPE = "_semantic_text_inference";
     public static final String NAME = "_semantic_text_inference";
-    public static final String SPARSE_VECTOR_SUBFIELD_NAME = "sparse_embedding";
-    public static final String TEXT_SUBFIELD_NAME = "text";
     public static final TypeParser PARSER = new FixedTypeParser(c -> new SemanticTextInferenceResultFieldMapper());
 
     private static final Map<List<String>, Set<String>> REQUIRED_SUBFIELDS_MAP = Map.of(
         List.of(),
-        Set.of(SPARSE_VECTOR_SUBFIELD_NAME, TEXT_SUBFIELD_NAME),
```

Comment:
The structure for the embeddings changes a bit. The field mapper was prepared to have an additional nesting level, but that is not required as the `asMap()` method from the results does not return the information on that format